src/dev-app/slide-toggle/slide-toggle-demo.html

```diff
@@ -14,9 +14,9 @@
 
   <p>Example where the slide toggle is required inside of a form.</p>
 
-  <form #form="ngForm" (ngSubmit)="onFormSubmit()" ngNativeValidate>
+  <form #form="ngForm" (ngSubmit)="onFormSubmit()">
 
-    <mat-slide-toggle name="slideToggle" required ngModel>
+    <mat-slide-toggle name="slideToggle" ngModel="[(formToggle)]">
```

Comment:
This doesn't look right.

Reply:
The syntax on the `ngModel` binding is wrong.

Reply:
ohh, that's not the right syntax 🤦 🍌 📦. I corrected the syntax, thanks.